server/__tests__/apollo-server.test.js

```diff
@@ -0,0 +1,76 @@
+import nock from "nock";
+import { gql } from "apollo-server";
+import { query } from "../test-utils/apollo-server";
+import breed from "../mocks/breed.json";
```

Comment:
Although there is a single breed, actually an array is stored. So maybe breedS is more appropriate here.